.evergreen/evergreen.yml

```diff
@@ -134,12 +134,13 @@ buildvariants:
 
   - name: generator
     display_name: Generator
-    allowed_requesters: ["github_pr"]
+    patch_only: true
     # Omit node_version expansion so that yarn-install fails to save time.
     run_on:
       - ubuntu2204-small
     tasks:
       - generate-tasks
+      - generate-spruce-e2e
```

Comment:
Can this be moved to the spruce variant? It will be an easier to select all in the Configure Page and I think it could prevent the error UX mentioned in the PR description.

Reply:
it wouldn't prevent the error because you'd still be generating the same exact task in the same exact variant (`e2e_parallel` task in `Spruce` variant), but I've modified it from Sophie's suggestion so that they're independent and I think it makes a little more sense now!